test/jdk/java/util/zip/ZipOutputStream/ZipOutputStreamMaxCenHdrTest.java

```diff
@@ -154,9 +168,8 @@ private static boolean writeZipEntry(ZipEntry zipEntry, boolean expectZipExcepti
         boolean receivedException = false;
         try (ZipOutputStream zos = new ZipOutputStream(
                 new BufferedOutputStream(Files.newOutputStream(ZIP_FILE)))) {
-            zos.putNextEntry(zipEntry);
             if (expectZipException) {
```

Comment:
Hello Lance, with this change, it looks like this now misses calling the `zos.putNextEntry(zipEntry)` when `expectZipException` is `false`. Is that an oversight?